jobs/integr8ly/poc-testing-executor.yaml

```diff
@@ -200,6 +200,7 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
 
             if (testPipeline == 'after-first-login-tests' || testPipeline == 'browser-based-testsuite-pipeline') {
                 parameters.add(string(name: 'EVALS_USERNAME', value: 'evals11@example.com'))
+                parameters.add(string(name: 'EVALS_PASSWORD', value: 'Password1'))
```

Comment:
I think you should additionally remove the EVALS_PASSWORD from the next 'if statement'